docs/core/tutorials/using-with-xplat-cli.md

```diff
@@ -147,6 +148,39 @@ Let's build off of the previous Fibonacci example by caching some Fibonacci valu
    377
    ```
 
+## Publish your app
+
+Once you're ready to distribute your app, you can publish it with the command [`dotnet publish`](../tools/dotnet-publish.md). Depending on the current .NET Core SDK you're using, the default [`dotnet publish`](../tools/dotnet-publish.md) command will choose a specific publishing method by default. For more information on what the default publishing method is for your SDK version, see [Publish .NET Core apps with the CLI](../deploying/deploy-with-cli.md) .
+
+There are two publishing methods:
+
+01. **Framework-dependent deployments** (FDD)\
```

Comment:
Can we get loc to change? `\` is a markdown tag, they should be adapting to it since its the writing system we use.

Also, the `01. ` is amazing and perfect because it creates your alignment for child content at 4-spaces instead of 3. :)